Cargo.toml

```diff
@@ -26,7 +26,7 @@ regex = "1"
 radix_trie = "0.2.1"
 
 [dev-dependencies]
-proxy-wasm-test-framework = { git = "https://github.com/Kuadrant/wasm-test-framework.git", branch = "kuadrant" }
+proxy-wasm-test-framework = { git = "https://github.com/Kuadrant/wasm-test-framework.git", branch = "fix-get-property-when-none" }
```

Comment:
this change will be reverted, only used for testing https://github.com/Kuadrant/wasm-test-framework/pull/2